### ISSUE TITLE
"msg": "[orapatch] module fail: Argument(s) Error... Patch base location is not valid.\n\n", "rc": 245

### ISSUE THREAD

**Issue by kiruba87** [iarsov/ansible-orapatch#1]
Hi

I am trying to use this module to patch 18c database present in the target machine, while I running the playbook i am getting the below error. Please check and help me to get this resolved. 

TASK [orapatch : Check OPatch minimum version] *************************************************************************************************************************
 [WARNING]: Module did not set no_log for root_password

failed: [104.45.3.126] (item={u'patch_db_list': u'', u'skip': False, u'run_only_checks': None, u'patch_id': 29708235, u'oracle_home_path': u'/mnt/resource/oracle/db_1', u'patch_db_all': True, u'debug': False, u'oracle_owner': u'oracle', u'patch_ojvm': True, u'patch_only_oh': False, u'host': None, u'oratab_file': None, u'backup_oh': False}) => {"ansible_loop_var": "item", "changed": false, "item": {"backup_oh": false, "debug": false, "host": null, "oracle_home_path": "/mnt/resource/oracle/db_1", "oracle_owner": "oracle", "oratab_file": null, "patch_db_all": true, "patch_db_list": "", "patch_id": 29708235, "patch_ojvm": true, "patch_only_oh": false, "run_only_checks": null, "skip": false}, "msg": "[orapatch] module fail: Argument(s) Error... Patch base location is not valid.\n\n", "rc": 245}

**Comment by iarsov** [comment]
Hi kiruba87,

Did you define patch details in patch_dict.yml? If yes, please post the definition of the patch.

**Comment by kiruba87** [comment]
I found the issue, 

In the orapatch_alert.log file, I could see the below error, 

Tue 13 Aug 2019 07:34:46 PM UTC command: /mnt/resource/oracle/db_1/OPatch/opatch prereq CheckMinimumOPatchVersion -phBaseDir /home/oracle/28822489
Tue 13 Aug 2019 07:34:47 PM UTC Module fail: Argument(s) Error... Patch base location is not valid.


Found that the patch is not unzipped at the target DB server, not sure whether this module will copy the patch binary to the target server and unzip it. I manually wrote a playbook to perform this step and patching completed without any issues. 

We can close this issue.
